src/baldr/incident_singleton.h

```diff
@@ -249,25 +283,24 @@ struct incident_singleton_t {
       auto wait = 0;
       if (latency > max_loading_latency) {
         LOG_WARN("Incident watcher is not meeting max loading latency requirement");
-      } // this round finished fast enough, changelog mode is cheap so wait only a second
+      } // this round finished fast enough wait the remainder of the time specified
       else {
-        wait = changelog ? 1 : max_loading_latency - latency;
```

Comment:
this seemed like a hack. we just listen to the configuration exactly, regardless of what mode its in